frontend/src/views/Apps/GeneralSettings/GeneralSettings.js

```diff
@@ -42,37 +40,15 @@ class GeneralSettings extends Component {
 
   async saveChanges() {
```

Comment:
can we encapsulate everything inside the save changes in a try/catch block?

Reply:
Currently the page only has a "showAlert" function which tells you that the Application has been saved.

Adding the try/catch is trivial but if it does catch an error, it should show some meaningful message to the user.  To do that, we'd need to add another "showError" function to pop up an error.

I think that is out of scope for this bug fix which is being pushed directly to master. But I could be wrong.

Reply:
Agree, please leave a Todo comment